amundsen_application/static/js/components/SearchPage/index.tsx

```diff
@@ -250,15 +252,24 @@ export class SearchPage extends React.Component<SearchPageProps, SearchPageState
       );
   };
 
+  renderTabs = () => {
```

Comment:
I think `renderContent` or `renderSearchResults` would be better names, since not every section here has tabs. This will also be where the default homepage content will be rendered as well.

Reply:
+1 for `renderContent`

Reply:
:+1: